src/main/resources/db/migration/0151/V193__ObservedSpeciesTotals.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE tracking.observed_plot_species_totals (
+    observation_id BIGINT NOT NULL REFERENCES tracking.observations ON DELETE CASCADE,
+    monitoring_plot_id BIGINT NOT NULL REFERENCES tracking.monitoring_plots ON DELETE CASCADE,
+    species_id BIGINT NOT NULL REFERENCES species,
+    total_live INTEGER NOT NULL DEFAULT 0,
+    total_dead INTEGER NOT NULL DEFAULT 0,
+    total_existing INTEGER NOT NULL DEFAULT 0,
+    total_plants INTEGER NOT NULL DEFAULT 0,
+    mortality_rate INTEGER NOT NULL DEFAULT 0,
+
+    PRIMARY KEY (observation_id, monitoring_plot_id, species_id)
+);
+
+CREATE INDEX ON tracking.observed_plot_species_totals (observation_id);
+CREATE INDEX ON tracking.observed_plot_species_totals (monitoring_plot_id);
+CREATE INDEX ON tracking.observed_plot_species_totals (species_id);
```

Comment:
Just curious if this could be a function with parameters, or would that be more complex and less readable? Not requesting changes.

Reply:
Thanks for the clarification, yes makes sense 👍